src/Order/OrdersList.php

```diff
@@ -107,7 +107,7 @@ public function add_order_barcode_column_content( $column, $order_id ) {
 	 * @return array.
 	 */
 	public function add_order_delivery_date_column_header( $columns ) {
-		$wc_actions = $columns['wc_actions'];
+		$wc_actions = isset( $columns['wc_actions'] ) ? $columns['wc_actions'] : '';
```

Comment:
This will not fix the issue, if `wc_actions` is not exist, then the code will add empty value [here](https://github.com/Progressus-io/postnl-for-woocommerce/blob/6fd38f6fe7fcfcd6dc74c1dbe4c030acee40d4a1/src/Order/OrdersList.php#L114) 

But we should know why this warning appeared in the first place, this function is a call back for 
`manage_edit-shop_order_columns` and `manage_woocommerce_page_wc-orders_columns` filters and 'wc_actions' should be exist, if they are not ( removed by other plugin or any changes by 3rd parties ) I think we can just add the PostNL actions to the array without unset/set the wc-actions.